app/controllers/calls_controller.rb

```diff
@@ -5,6 +5,9 @@ def create
     @call = c.calls.new(status: params[:status])
 		if @call.save
 			redirect_to root_path
+    else
+      flash[:alert] = "Call failed to save! Please submit the call again."
```

Comment:
You totally have the right idea here -- tests are a great opportunity to refactor and add stuff like this, so if something goes wrong you have a better sense of why. (At least, that's my own philosophy of testing!)